src/selectedtests/task_mappings/mappings.py

```diff
@@ -265,6 +274,29 @@ def _filter_non_matching_distros(builds: List[Build], build_regex: Pattern) -> L
     return [build for build in builds if match(build_regex, build.display_name)]
 
 
+def _process_evg_version(
```

Comment:
This is pretty much just a wrapper function that is being used for the worker function.